agent/bench-scripts/pbench-run-benchmark

```diff
@@ -360,8 +369,11 @@ while (scalar @param_sets > 0) {
         $iteration_id++;
     }
     if ($pp_only) {
-        print BULK_FH "echo Waiting for $nr_pp_jobs post-processing jobs to finish\n";
-        print BULK_FH "wait\n";
+	if ($params{'postprocess-mode'} eq 'cdm') {
```

Comment:
Seems like a tabs vs spaces formatting disagreement here.

Reply:
I'm not an expert in this area...but isn't this type of editor behavior supposed to be controlled by these lines at the top of the files:

```
#!/usr/bin/perl
# -*- mode: perl; indent-tabs-mode: t; perl-indent-level: 4 -*-
# vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=perl
```

If so, does that mean these not correct?

Reply:
It looks like we have a fight going on with tabs and spaces, where we have tabs in the file, but have `expandtab` in the `vim` settings, but don't have that specified in the `emacs` settings.

I have opened issue #1500 to propose a path forward to fix this.